documentation/general/dash-sonic-hld.md

```diff
@@ -326,10 +337,11 @@ key                      = DASH_ROUTE_TABLE:eni:prefix ; ENI route table with CA
 action_type              = routing_type              ; reference to routing type
 vnet                     = vnet name                 ; destination vnet name if routing_type is {vnet, vnet_direct}
 appliance                = appliance id              ; appliance id if routing_type is {appliance} 
-overlay_ip               = ip_address                ; overlay_ip to override if routing_type is {servicetunnel}, overly_ip to lookup if routing_type is {vnet_direct}, use dst ip from packet if not specified
-underlay_ip              = ip_address                ; underlay_ip to override if routing_type is {servicetunnel}, use dst ip from packet if not specified
-overlay_sip              = ip_address                ; overlay_sip if routing_type is {servicetunnel}  
-underlay_sip             = ip_address                ; overlay_sip if routing_type is {servicetunnel}
+overlay_ip               = ip_address                ; overly_ip to lookup if routing_type is {vnet_direct}, use dst ip from packet if not specified
+overlay_sip              = ip_address                ; overlay ipv6 src ip if routing_type is {servicetunnel}, transform last 32 bits from packet (src ip)
+overlay_dip              = ip_address                ; overlay ipv6 dst ip if routing_type is {servicetunnel}, transform last 32 bits from packet (dst ip) 
+underlay_sip             = ip_address                ; underlay ipv4 src ip if routing_type is {servicetunnel}; this is the ST VIP
```

Comment:
Can we assume these are always provided for service tunnel configs?

Reply:
Yes